build.gradle

```diff
@@ -78,24 +64,28 @@ task functionalTestsFirefox(type: NodeTask) {
   args = ['--replay-fixtures', '--serve-deck', '--imposter-port=8084', '--browser=firefox', '--headless']
 }
 
-task generateVersionFile << {
-  'git update-index --assume-unchanged version.json'.execute()
-  def buildInfo = [
-    version: project.hasProperty('deckVersion') ? "${deckVersion}" : "n/a",
-    created: new Date().getTime()
-  ]
-  def buildJson = JsonOutput.prettyPrint(JsonOutput.toJson(buildInfo))
-  new File(Paths.get("build", "webpack").toString()).mkdirs()
-  new File(Paths.get("build", "webpack", "version.json").toString()).write(buildJson)
-  new File("version.json").write(buildJson)
+task generateVersionFile {
+  doLast {
+    'git update-index --assume-unchanged version.json'.execute()
+    def buildInfo = [
+      version: project.hasProperty('deckVersion') ? "${deckVersion}" : "n/a",
+      created: new Date().getTime()
+    ]
+    def buildJson = JsonOutput.prettyPrint(JsonOutput.toJson(buildInfo))
+    new File(Paths.get("build", "webpack").toString()).mkdirs()
+    new File(Paths.get("build", "webpack", "version.json").toString()).write(buildJson)
+    new File("version.json").write(buildJson)
+  }
 }
 
 webpack.outputs.dir file('build/webpack')
 
 yarn.dependsOn 'generateVersionFile'
 webpack.dependsOn 'yarn'
 karma.dependsOn 'webpack'
-test.dependsOn 'karma'
+project.tasks.register('test') {
```

Comment:
`test` lifecycle hook doesn't exist because we aren't applying java anymore, so creating it here to tie in to running karma